tennis-scheduler-web-app/src/components/Registration.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState, useEffect } from "react"
+import { useNavigate, Link } from 'react-router-dom'
+
+export const Registration = () => {
+    const initialValues = { name: "", surname: "", email: "", password: "", confirmPassword: "" }
+    const [formValues, setFormValues] = useState(initialValues);
+    const [formErrors, setFormErrors] = useState({});
+    const [isSubmit, setIsSubmit] = useState(false);
+
+    const handleChange = e => {
+        const { name, value } = e.target;
+        setFormValues({ ...formValues, [name]: value });
+    };
+
+    const registrationSubmit = e => {
+        e.preventDefault();
+        setFormErrors(validation(formValues));
+        setIsSubmit(true);
+    };
+
+    useEffect(() => {
+        if (Object.keys(formErrors).length === 0 && isSubmit) {
+            console.log(formErrors);
+        }
+    }, [formErrors]);
```

Comment:
It is always better to inform users of error that occurred than console log it.

Reply:
I deleted this useEffect because I used it for debugging, it's not necessary to be here. This forms errors are listed in paragraphs under the fields.